sp800-63-3/sec1_2_introduction.md

```diff
@@ -68,21 +68,18 @@ their authentication assurance requirements:
     level* - As some implementations may create or compound particular
     risks, agencies should conduct a final validation to confirm that
     the system achieves the required assurance level for the
-    user-to-agency process. NIST SP 800-53A [[SP 800-53A]](#SP800-53A)
+    user-to-agency process. NIST [[SP 800-53A]](https://pages.nist.gov/800-63-3/sp800-63-3.html#SP800-53A)
```

Comment:
This reference doesn't need to be fully qualified if `#SP800-53A` is in the -3 references doc.